README.md

```diff
@@ -2,51 +2,51 @@
 
 # Kotlin/Native  #
 
-_Kotlin/Native_ is a LLVM backend for the Kotlin compiler, runtime
-implementation and native code generation facility using LLVM toolchain.
+_Kotlin/Native_ is an LLVM backend for the Kotlin compiler, runtime
```

Comment:
an?

Reply:
Yes, when abbreviating to an acronym, you are meant to put the correct article relative to the letters of that acronym. Since L is pronounced “el,” the appropriate article is “an.”

Reply:
Cool, thanks!